src/services/dashboards/widgets/_components/WidgetDataTable.vue

```diff
@@ -239,6 +267,13 @@ export default defineComponent<Props>({
             }
             return item;
         };
+        const getHandler = (option: Field['icon']|Field['link']|Field['rapidIncrease'], item) => {
```

Comment:
This handler's return type looks like little bit confusing. How about adding return type?